File: day_1/day_1_1.rb
the_most_calories = File.read('input.txt')
                        .split(/\n\n/)
                        .map do |group|
  group
    .strip
    .split(/\n/)
    .map(&:to_i)
end
                        .map(&:sum)
                        .max

puts the_most_calories

File: day_1/day_1_1.rs
use std::fs;

fn main() {
    let the_most_calories: i32 = fs::read_to_string("input.txt")
        .unwrap()
        .split("\n\n")
        .map(|group| {
            group
                .trim()
                .split("\n")
                .map(|line| line.parse::<i32>().unwrap())
        })
        .map(|numbers| numbers.sum())
        .max()
        .unwrap();

    print!("{}\n", the_most_calories);
}
